examples/rllib/view_models.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2020 DeepMind Technologies Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Runs the bots trained in self_play_train.py and renders in pygame.
+You must provide experiment_state, expected to be
+~/ray_results/PPO/experiment_state_YOUR_RUN_ID.json
+"""
+
+import dm_env
+from dmlab2d.ui_renderer import pygame
+import numpy as np
+from ray.rllib.agents.registry import get_trainer_class
+from ray.tune.analysis.experiment_analysis import ExperimentAnalysis
+from ray.tune.registry import register_env
+
+from examples.rllib.utils import env_creator
```

Comment:
Like in the other file, please import the module, and acces its properties/objects from there.